File: app/src/main/java/coolpharaoh/tee/speicher/tea/timer/views/exportimport/datatransfer/pojo/TeaPOJO.java
package coolpharaoh.tee.speicher.tea.timer.views.exportimport.datatransfer.pojo;

import java.util.Date;
import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class TeaPOJO {
    private String name;
    private String variety;
    private int amount;
    private String amountKind;
    private int color;
    private int nextInfusion;
    private Date date;
    private List<InfusionPOJO> infusions;
    private List<CounterPOJO> counters;
    private List<NotePOJO> notes;
}

File: app/src/main/java/coolpharaoh/tee/speicher/tea/timer/views/export_import/data_transform/pojo/TeaPOJO.kt
package coolpharaoh.tee.speicher.tea.timer.views.export_import.data_transform.pojo

import java.util.Date

class TeaPOJO {
    var name: String? = null
    var variety: String? = null
    var amount: Double = 0.0
    var amountKind: String? = null
    var color: Int = 0
    var rating: Int = 0
    var inStock: Boolean = false
    var nextInfusion: Int = 0
    var date: Date? = null
    var infusions: List<InfusionPOJO>? = null
    var counters: List<CounterPOJO>? = null
    var notes: List<NotePOJO>? = null
}